### PR TITLE
Fix ConfigMap not found errors after deletion

### PR DESCRIPTION
**Pull request by mnitchev**
Since reconciliation takes a while per request, this PR also increases concurrency

## Checklist

- [x] Update changelog in CHANGELOG.md.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,21 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
 
+### Added
+
+- Add option to configure controller concurrency for CAPA and EKS.
+
+### Fixed
+
+- Fix ConfigMap not found errors after deletion is done.
+
+## [0.27.7] - 2024-06-25
```
Comment (fiunchinho) [inline CHANGELOG.md:18]: This looks wrong

Comment (mnitchev) [reply CHANGELOG.md:18]: fixed I think

File: controllers/capa_controller.go [hunk 2]
```diff
@@ -79,14 +80,23 @@ func (r *CAPAClusterReconciler) Reconcile(ctx context.Context, req ctrl.Request)
...
+
 	cluster, err := util.GetClusterFromMetadata(ctx, r.Client, awsCluster.ObjectMeta)
 	if err != nil {
 		return reconcile.Result{}, microerror.Mask(err)
 	}
 
-	if annotations.HasPaused(awsCluster) {
-		logger.Info("AWSCluster is marked as paused, skipping")
-		return ctrl.Result{}, nil
+	// If the cluster is already deleted the configmap and role identity will
```
Comment (fiunchinho) [inline controllers/capa_controller.go:93]: The role identity is shared among different WCs, right? I don't think it's removed when removing a cluster.

**Review by fiunchinho — APPROVED** [review]

File: controllers/capa_controller.go (hunk 2 above)
Comment (mnitchev) [reply controllers/capa_controller.go:93]: yeah you're right. I guess I was thinking of the acceptance tests in resolver-rules-operator where it's removed after each test

**Pull request merged**